q2_types/feature_data/_transformer.py

```diff
@@ -227,6 +246,19 @@ def _read_dna_fasta(path):
     return skbio.read(path, format='fasta', constructor=skbio.DNA)
 
 
+def _dnafastaformats_to_series(ff):
+    data = {}
+    for sequence in _read_dna_fasta(str(ff)):
+        data[sequence.metadata['id']] = sequence
+    return pd.Series(data)
+
+
+def _dnafastaformats_to_metadata(ff):
+    df = _dnafastaformats_to_series(ff).to_frame()
```

Comment:
Can you have the dataframe store sequence strings instead of `skbio.DNA` objects?